tor/tor-service-defaults-torrc

```diff
@@ -0,0 +1,15 @@
+DataDirectory /var/lib/tor
```

Comment:
This is the default right? Why do we have to provide it then?

Reply:
There are a few things about this file that I don't fully understand the reason for. This is the created from the default set up when installing Tor on Ubuntu 18. I elected to just leave it as is instead of deleting things that I didn't see why they put there in the first place. I don't actually see why Tor uses this file at all and not just the torrc - see why I was hesitant to change it :)

Reply:
I have removed the whole `tor/` directory now.